src/movegen.c

```diff
@@ -358,8 +345,8 @@ int squareIsAttacked(Board* board, int colour, int sq){
 
     return    (                pawnAttacks(sq, enemyPawns, colour)          )
```

Comment:
Line the parenthesis back up, or just remove my silly formatting

Reply:
remove silly formatting